Common/Gfx/Shape.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#ifndef RM_GFX_SHAPE_H_
+#define RM_GFX_SHAPE_H_
+#include <gdiplus.h>
+namespace Gfx {
+	struct Shape
+	{
+		Shape() :
+			m_FillColor(Gdiplus::Color::White),
+			m_OutlineWidth(1),
+			m_OutlineColor(Gdiplus::Color::Black)
+		{}
+
+		Microsoft::WRL::ComPtr<ID2D1Geometry> m_Shape;
+		Gdiplus::Color m_FillColor;
```

Comment:
Instead of storing `Gdiplus::Color` and then using `ToColorF`, could you just store a `D2D1_COLOR_F` directly?